src/components/AdminDashboard/AdminDashboard.js

```diff
@@ -189,129 +146,103 @@ const AdminDashboard = () => {
     setAdding(true);
   };
 
-  const logout = () => {
-    localStorage.removeItem('okta-token-storage', 'okta-cache-storage');
-    push('/');
-    window.location.reload();
+  // this needs to be improved
+  const selectedTabButtonStyle = {
+    background: '#095fab'
   };
+
   return (
     <>
+      {/* I don't know what this is */}
       {showModal ? <div className="back-drop"></div> : null}
       <Modal
         showModal={showModal}
         modalHandler={modalHandler}
-        unapprovedIncidents={unapprovedIncidents}
+        unapprovedIncidents={pendingIncidents}
       />
 
+      {/* Incident "tabs" - unapproved, approved, form responses */}
       <div className="dashboard-buttons-container">
         <div className="incident-btn-container">
           <button
             className="approve-btn"
-            onClick={() => setListType('unapproved')}
+            style={listType === 'pending' ? selectedTabButtonStyle : {}}
+            onClick={() => setListType('pending')}
           >
-            Unapproved Incidents
+            Pending Incidents
           </button>
           <button
             className="approve-btn"
+            style={listType === 'approved' ? selectedTabButtonStyle : {}}
             onClick={() => setListType('approved')}
           >
             Approved Incidents
           </button>
           <button
             className="approve-btn"
+            style={listType === 'form-responses' ? selectedTabButtonStyle : {}}
             onClick={() => setListType('form-responses')}
           >
             Form Responses
           </button>
         </div>
-        <div className="logout-container">
-          <button className="approve-btn" onClick={logout}>
-            Log Out
-          </button>
-        </div>
       </div>
-      {listType === 'unapproved' && (
-        <div className="dashboard-container">
-          <DashboardTop
-            unapprovedIncidents={unapprovedIncidents}
-            toggleAddIncident={toggleAddIncident}
-            listType={listType}
+
+      <div className="dashboard-container">
+        <DashboardTop
+          unapprovedIncidents={pendingIncidents}
+          toggleAddIncident={toggleAddIncident}
+          listType={listType}
+        />
+
+        {selectedIds.length > 0 &&
+          <span>{selectedIds.length} selected</span>
+        }
+
+        {/* Controls for setting the status of selected incidents (unapproved, pending, approved) */}
+        <IncidentStatus
+          isActive={selectedIds.length > 0}
+          listType={listType}
+          onStatusConfirm={approveAndRejectHandler}
+        />
+
+        {/* modal popup for adding a new incident */}
+        {adding &&
+          <AddIncident
+            getData={getData}
+            setAdding={setAdding}
           />
+        }
 
-          {adding ? (
-            <AddIncident
-              setPageNumber={setPageNumber}
```

Comment:
So without this, the ability to create an incident from any list view is lost, unless you've completely separated that code from every listview. I'll await the loom video to see this functionality re-created.

Reply:
It's actually still there, git just didn't line it up right. (see line 211 in the updated file).